apps/api/src/modules/payments/payments.service.ts

```diff
@@ -762,29 +762,23 @@ export default class PaymentsService {
     return matchingPayments
   }
 
-  async getPaymentById(paymentId: string) {
+  async getPaymentById(paymentId: string, isAdmin?: boolean) {
     const payment = await PaymentModel.query().findById(paymentId)
     userInvariant(payment, 'payment not found', 404)
-    return payment
-  }
-
-  async getAdminPaymentById(paymentId: string) {
-    const payment = await PaymentModel.query()
-      .findById(paymentId)
-      .withGraphFetched('pack')
-    userInvariant(payment, 'payment not found', 404)
-    const { pack } = payment
-    if (pack.templateId) {
-      const { packs: packTemplates } = await this.packs.getPublishedPacks({
-        templateIds: [pack.templateId],
-      })
-      const packTemplate = packTemplates[0]
-      return {
-        ...payment,
-        pack: {
-          ...packTemplate,
-          ...pack,
-        },
+    if (isAdmin) {
+      const { pack } = payment
+      if (pack.templateId) {
+        const { packs: packTemplates } = await this.packs.getPublishedPacks({
+          templateIds: [pack.templateId],
+        })
+        const packTemplate = packTemplates[0]
+        return {
+          ...payment,
+          pack: {
+            ...packTemplate,
+            ...pack,
+          },
+        }
```

Comment:
@afraser I also pushed 68205f3 with another update we discussed, passing in a query parameter to determine what gets returned by the `getPaymentById` method, removing the extra method, and cleaning up mentions of "admin" in endpoint names.

Reply:
Still working on this revoke endpoint though, and planned to merge this PR into my other PR when complete, but let me know if I'm holding you up in any way and I can get these changes into the other branch.

Reply:
@dallashuggins no hold up here. Your last PR came at the perfect time so I'm currently working on filling in the transaction show page with real data and styles.